package com.self.Exceptions

class Exception1 {

	try{
	  remoteCalculatorService()
	}
}